usecases/auth/authorization/conv/casbin_types.go

```diff
@@ -206,14 +186,27 @@ func policy(permission *models.Permission) (*authorization.Policy, error) {
 		resource = CasbinRoles(role)
 	case authorization.ClusterDomain:
 		resource = CasbinClusters()
-	case authorization.SchemaDomain:
+	case "collections":
+		domain = authorization.SchemaDomain
 		collection := "*"
-		tenant := "*"
+		tenant := "#"
 		if permission.Collections != nil && permission.Collections.Collection != nil {
 			collection = schema.UppercaseClassName(*permission.Collections.Collection)
 		}
-		if permission.Collections != nil && permission.Collections.Tenant != nil {
-			tenant = *permission.Collections.Tenant
+		resource = CasbinSchema(collection, tenant)
+
+	case "tenants":
+		domain = authorization.SchemaDomain
```

Comment:
why this change? Or specifically, why are we not having a separate tenant domain?

Reply:
because the endpoint params , doesn't have to be exposed to internal casbin , this step is the mapping between endpoint and casbin internal 
`Domain` in general at the moment has no use  from casbin internal PoV but later one it could come handy 

tenants and collection both are schema domain action and they shall be we don't need an extra domain